etl/steps/data/garden/health/2023-04-18/shared.py

```diff
@@ -0,0 +1,688 @@
+            "3": "Not at all comfortable",
+            "99": "DK/Refused",
+        },
+    },
+    "mh6": {
+        "title": "Friends/family have been anxious/depressed",
+        "answers": {
+            "1": "Yes",
+            "2": "No",
+            "99": "DK/Refused",
+        },
+    },
+    "mh7a": {
+        "title": "Have been anxious/depressed",
+        "answers": {
+            "1": "Yes",
+            "2": "No",
+            "99": "DK/Refused",
+        },
+    },
+    "mh7b": {
+        "title": "Age when first felt anxious/depressed",
+        "answers": {},
+    },
+    "mh7b_2": {
+        "title": "Age range when first felt anxious/depressed",
+        "answers": {
+            "1": "Ages <13",
+            "2": "Ages 13-19",
+            "3": "Ages 20-29",
+            "4": "Ages 30-39",
+            "5": "Ages ≥40",
+            "99": "DK/Refused",
+        },
+    },
+    "mh7c": {
+        "title": "Have felt anxious/depressed more than once",
+        "answers": {
+            "1": "Yes",
+            "2": "No",
+            "99": "DK/Refused",
+        },
+    },
+    "mh8a": {
+        "title": "Talked to mental health professional when anxious/depressed",
+        "answers": {
+            "1": "Yes",
+            "2": "No",
+            "99": "DK/Refused",
+        },
+    },
+    "mh8b": {
+        "title": "Engaged in religious/spiritual activities when anxious/depressed",
+        "answers": {
+            "1": "Yes",
+            "2": "No",
+            "99": "DK/Refused",
+        },
+    },
+    "mh8c": {
+        "title": "Talked to friends or family when anxious/depressed",
+        "answers": {
+            "1": "Yes",
+            "2": "No",
+            "99": "DK/Refused",
+        },
+    },
+    "mh8d": {
+        "title": "Took prescribed medication when anxious/depressed",
+        "answers": {
+            "1": "Yes",
+            "2": "No",
+            "99": "DK/Refused",
+        },
+    },
+    "mh8e": {
+        "title": "Improved healthy lifestyle behaviors when anxious/depressed",
+        "answers": {
+            "1": "Yes",
+            "2": "No",
+            "99": "DK/Refused",
+        },
+    },
+    "mh8f": {
+        "title": "Made a change to work situation when anxious/depressed",
+        "answers": {
+            "1": "Yes",
+            "2": "No",
+            "99": "DK/Refused",
+        },
+    },
+    "mh8g": {
+        "title": "Made a change to personal relationships when anxious/depressed",
+        "answers": {
+            "1": "Yes",
+            "2": "No",
+            "99": "DK/Refused",
+        },
+    },
+    "mh8h": {
+        "title": "Spent time in nature/the outdoors when anxious/depressed",
+        "answers": {
+            "1": "Yes",
+            "2": "No",
+            "99": "DK/Refused",
+        },
+    },
+    "mh9a": {
+        "title": "How helpful was talking to mental health professional when anxious/depressed",
```

Comment:
Now this one is in direct form?

Reply:
Yeah... Perhaps making it indirect can be confusing because the sentence is very long.